src/Modal/Modal.component.tsx

```diff
@@ -31,10 +32,12 @@ const StyledModal = StyledReactModal.default.styled`
     flex-direction: column;
     justify-content: space-between;
 
-    border-radius: modal;
-    ${({ background = 'neutrals.white.base' }: ModalProps) =>
-        css({ backgroundColor: background })};
-    ${({ color }: ModalProps) => css({ color })}
+    ${({ background = 'neutrals.white.base', color }: ModalProps) =>
+        css({
+            background: th.color(background),
+            color,
+            borderRadius: 'modal',
```

Comment:
This component is defined like so `StyledReactModal.default.styled` so it doesn't run through the xstyled interpolation. I moved the borderRadius prop to the xstyled `css` function so that it is parsed as expected.